test/test_cloudformation_utils.py

```diff
@@ -489,3 +490,47 @@ def test_abstract_orchestration_manager_discover_foursight_check_runner_name():
                     assert foo_runner == MOCKED_DEV_CHECK_RUNNER
                     assert os.environ.get('CHECK_RUNNER') == foo_runner
                     print(f"CHECK_RUNNER={os.environ.get('CHECK_RUNNER')}")
+
+
+def test_find_lambda_function_names():
+
+    mocked_lambdas = [
+        "c4-foursight-cgap-supertest-stack-CheckRunner-ABC",
+        "c4-foursight-fourfront-production-stac-CheckRunner-DEFGHI",
```

Comment:
You should insert about 50 generated 'decoy' items in each position before, amidst, and after these so that you're exercising all the nextmarker stuff, forcing things onto different pages.

Reply:
Oh yeah good call, and trying it actually running into something, thanks.

Reply:
Added a bunch of dummy lambdas to the list (total: 203) to test the nextmarker stuff.